src/openforms/variables/admin.py

```diff
@@ -0,0 +1,17 @@
+from django.contrib import admin
+
+from .models import ServiceFetchConfiguration
+
+
+@admin.register(ServiceFetchConfiguration)
+class ServiceFetchConfigurationAdmin(admin.ModelAdmin):
+    raw_id_fields = [
+        "service",
+    ]
+    list_display = [
+        "service",
+        "path",
+    ]
+    list_filter = [
+        "service",
+    ]
```

Comment:
1. let's add `method` to the `list_display` too
2. `search_fields` should also be defined, and it should search through the `path` and `service__api_root`

Reply:
:slightly_smiling_face:  I had `method` in there, but removed it because in my happy place there should only be `GET`s in there. :see_no_evil: